### ISSUE TITLE
Extract data on releases from GitHub and CRAN

### ISSUE THREAD

**Issue by mpadge** [ropensci-org/makeregistry#76]

**Comment by maelle** [comment]
what's the context for this? :smile_cat:

**Comment by mpadge** [comment]
https://github.com/orgs/ropensci/projects/9?pane=issue&itemId=45288442 - "Do more to version-stamp peer review". We ultimately want records of at least version submitted + version approved, yet we currently don't have that information. This is a step towards ensuring release details are automatically recorded.

**Comment by maelle** [comment]
@mpadge do you get the data you need from R-universe's API?

**Comment by mpadge** [comment]
No, because we still don't extract or store those data in the registry at all. Do you have a use case for them?

**Comment by maelle** [comment]
Not that I know of?
